README.md

```diff
@@ -8,6 +8,38 @@ To do this run `make run-validator` to run the validator and then run `make vali
 
 **Schema file names must use snake case to be compatible with runner**
 
+## Generating translated schemas
+
+** N.B. Currently only supporting translations for Health themed surveys (PHM).
+
+Before running any translations script it is important to check that the latest release of the eq-translations repo is being
+used. This can be done using:
+```
+make translations-check
+```
+
+To generate a translation template `.pot` file in order to translate a schema, use the following command. It will generate a template file containing all the strings to be translated:
+```
+make translation-templates
+```
+
+To translate a schema, a `.po` file for the schema will need to be added to the `translations/{THEME}/{LANGUAGE_CODE}` directory containing strings from the schema
+and the matching translations. 
+
+The `.po` file needs to be named in the following format: `{SCHEMA_NAME}_{LANGUAGE_CODE}.po`
+
+Once this in place, the following command can be run in order to generate a translated
+schema:
+```
+make translate-schemas
+```
+The translated schema will be added to the `/schemas/{THEME}/{LANGUAGE_CODE}` directory.
+
+To check that translations are up to date use the following command (This check will run automatically when a pull request is raised):
+```
+make test-translate-templates
+```
```

Comment:
I would title these for clarity.
- Generating Translation Templates
- Generating Translated Schemas
- Testing Translation Templates